config.yaml.defaults

```diff
@@ -61,6 +61,8 @@ app:
       component: UserManagement
     - path: "/upload_spectrum/:id"
       component: UploadSpectrum
+    - path: "/finder/:id"
```

Comment:
Should this be `/source/:id/finder` and then specify the correct query parameter below in the URL?